synapse/tests/test_lib_stormhttp.py

```diff
@@ -544,6 +546,64 @@ async def test_storm_http_proxy(self):
             self.false(resp.get('ok'))
             self.isin('connect to proxy 127.0.0.1:1', resp['mesg'])
 
+        async with self.getTestCore() as core:
```

Comment:
Should this have a `conf = {'http:proxy': 'socks5://user:pass@127.0.0.1:1'}` value?